crates/types/src/image.rs

```diff
@@ -644,4 +790,21 @@ mod tests {
             deserialize(&serialize(&image).unwrap()).unwrap();
         assert_eq!(deserialized_serialized_image, image);
     }
+
+    #[test]
+    fn image_raw_can_deserialize_image_data() {
+        let image = Image {
+            width_422: 1,
+            height: 1,
+            buffer: Arc::new(vec![YCbCr422 {
+                y1: 0,
+                cb: 1,
+                y2: 2,
+                cr: 3,
+            }]),
+        };
+
+        let data = bincode::serialize(&image).unwrap();
```

Comment:
Be consistent and just use `serialize` it is already imported and used above.

Reply:
Done